data/kahet/kahet missions.txt

```diff
@@ -721,3 +721,1128 @@ event "pacified ka'sei"
+			choice
+				`	"Thank you."`
+					goto skipdialog
+				`	"Can you tell me more about that?"`
+			`	"It seems their atmosphere may have been similar to ours - the air in the settlement we visited was almost evenly split between oxygen and argon: breathable by human standards."`
+			`	"We could have removed our helmets after the airlock?" you ask.`
+			`	"It is a bit more complicated than that. We found that the pressure inside the complex was much, much lower than what we are used to - less than a sixteenth of an atmosphere. On most Earth-like planets, you would only find such pressures at over 20,000 meters. There are two possibilities: either the Builders were born in a thin atmosphere, or the life support system failed some time ago and the structure has been slowly losing air ever since. We hope to find the answer soon enough."`
+
+			label skipdialog
+			`	"On top of that, we have learned that the Fetri'sei that attacked us was not waiting in the structure when we first entered the system; it was gathering materials from the asteroid belt together with the rest of its fellow Ka'sei, and only moved inside the dome when they spotted us."`
+
+			label proposal
+			choice
+				`	"What are you suggesting?"`
+			`	"Cloaking is the key. If we can stay hidden as we enter Patir and land, then the Fetri'sei should stay outside of the settlement, letting us explore further. Make sure your ship has enough fuel for the journey, however; if you drop your cloak at any time in Patir, then the Ka'sei will block our progress again."`
+			`	After he finishes signing, he makes a few quick gestures toward the projector, enlarging the hologram of the Builder settlement and revealing what looks like a flight path. "You are going to land roughly where we did last time, carrying me and four other xenoarcheologists. If you accept, I should be able to allocate 100,000 credits to help you buy the necessary equipment for this mission."`
+			choice
+				`	"I am in."`
+					accept
+				`	"What if they detect us while inside the structure?"`
+				`	"Sorry, but I do not think it is a good idea to return there."`
+					decline
+			`	"It is a possibility, yes. It is also quite problematic because communication between the inside and the outside of the structure is impossible. I think the best course of action would be to place a scanner that controls the large airlocks on the upper side of the dome, the same ones the Ka'sei used last time. If I am right, it should give us around a quarter of an hour to get back in the airlock and escape."`
+			choice
+				`	"Alright, let's do it."`
+					accept
+				`	"Sorry, but I do not think it is a good idea to return there."`
+					decline
+
+	on accept
+		payment 100000
+		set "remnant: dusk busy"
+	on abort
+		# Do nothing.
+	on fail
+		dialog
+			`As soon as you approach the <planet>, you see that the Ka'sei are following your ship. You won't be able to enter the settlement safely anymore.`
```

Comment:
> Also, why are you approaching the planet?

...Because the settlement is on it? Also, this suggestion changes "As soon as you approach the Builder Settlement," to "As soon as you approach Builder Settlement".

Reply:
In my opinion, arriving in the system for the mission can be considered "approaching the settlement" even if you haven't explicitly started the landing itself, so it should work either way. I think "the builder settlement" flows better here, but i don't mind either way